### PR TITLE
Richard_FizzBuzz

### PR DESCRIPTION
**Pull request by RichardSeverich**

### CODE REVIEW

File: src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public boolean isBuzz(int number) {
+        return number % 5 == 0 || String.valueOf(number).contains("5");
+    }
+
+    /**
+     * @param number contains of number value.
+     * @return Fizz, Buzz or FizzBuzz.
+     */
+    public String getFizzBuzz(int number) {
+        StringBuilder result = new StringBuilder();
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java]: Do we really need a StringBuilder here ?

File: src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ * Created by Administrator on 8/25/2017.
+ */
+public class FizzBuzz {
+
+    /**
+     * @param number is the number.
+     * @return boolean value.
+     */
+    public boolean isFizz(int number) {
+        return number % 3 == 0 || String.valueOf(number).contains("3");
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java]: Please add constant for 3 and 5
also for Fizz and Buzz

File: src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java (hunk 1 above)
Comment (RichardSeverich) [reply src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java]: Not really, I'll solve that problem.

File: src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java (hunk 2 above)
Comment (RichardSeverich) [reply src/main/java/org/fundacionjala/coding/richard/FizzBuzz.java]: Okay, I'll solve that problem.

**Comment by carledriss** [comment]
LGTM!

**Pull request merged**